addons/binding/org.openhab.binding.milight/README.md

```diff
@@ -1,92 +1,137 @@
+
+| Bulb Type          | Milight Bridge | iBox  | iBox2 |
+|--------------------|:--------------:|:-----:|:-----:|
+| Dual White         |       ✓        |       |       |
+| RGB/White          |       ✓        |       |       |
+| 2016 Dual White    |                |   ✓   |   ✓   |
+| 2016 RGB/White     |                |   ✓   |   ✓   |
+| RGB/Cold,Warmwhite |                |   ✓   |   ✓   |
+
+Please note that LD382, LD382A, LD686 RGB stripes and bulbs are supported by the 
+[WifiLed Binding](https://github.com/openhab/openhab2-addons/blob/master/addons/binding/org.openhab.binding.wifiled/README.md).
 
 ## Discovery
-Version 3+ bridges can be discovered by triggering a search in openHAB's inbox. Found bridges
-will show up an can easily be added as things.
-After a bridge has been added, all possible bridge supported devices will appear
-as new things. Unfortunatelly milight leds are only able to receive so there is
-no real auto detection for single milight leds but only for the briges possible.
-Add the leds you actually configured and hide the rest of the detected things.
+All supported bridges can be discovered by triggering a search in openHAB's Inbox. Found bridges
+will show up and can easily be added as things.
+Unfortunately Milight like bulbs have no back channel and can not report their presence, therefore
+all possible bulbs are listed as new things after a bridge has been added.
+Add the bulbs you actually configured and hide the rest of the detected things.
 
-## Binding Configuration
-When manually adding an older bridge Type (3-), you have to add configuration information for
-the bridge IP-Address and the listening port.
+Your device needs to be connected to your local network (i.e. by using the WPS button connection method or the native App shipped with the device). Read the device manual for more information about how to connect your device to your network.
 
 ## Thing Configuration
 Besides adding bridges through Paper-UI, you can also add them manually in your Thing
-configuration file.
-
-    Bridge milight:bridge:ACCF23A6C0B4 [ ADDR="192.168.0.70", PORT=8899 ] {
-	    Thing whiteLed 0
-	    Thing whiteLed 1
-	    Thing rgbLed 5
-	    Thing rgbLed 8
-	    Thing rgbLed 9
-	    Thing rgbLed 10
+configuration file. iBox and iBox2 have the version 6, older milight bridges have the version 3.
+The ID is the MAC address of the bridge in hexadecimal digits.
+
+    Bridge milight:bridge:ACCF23A6C0B4 [ ADDR="192.168.0.70", CONFIG_PROTOCOL_VERSION=3, ID="ACCF23A6C0B4" ] {
+        Thing whiteLed 0
+        Thing rgbwwLed 4
+        Thing rgbLed 1
     }
 
 The Thing configuration for the bridge uses the following syntax
-Bridge milight:bridge:<mac address of bridge> [ ADDR="<IP-Address of bridge>", PORT=<listening port> ]
+Bridge milight:bridge:<mac address of bridge> [ ADDR="<IP-Address of bridge>", CONFIG_PROTOCOL_VERSION=3/6 ID=<mac> ]
```

Comment:
Shouldn't this be
Bridge milight:bridgeV6:<mac address of bridge>
?

Same in the example above. I'm not sure if CONFIG_PROTOCOL_VERSION is still used then?